serverlessworkflow/modules/ROOT/pages/cloud/operator/using-persistence.adoc

```diff
@@ -0,0 +1,107 @@
+= Building a Custom Development Image
+:compat-mode!:
+// Metadata:
+:description: Using persistence in the workflow instance to store its context
+:keywords: sonataflow, workflow, serverless, operator, kubernetes, persistence
+
+
+This document describes how to configure a SonataFlow instance to use persistence to store the flow's context in a relational database.
+
+== Configuring the SonataFlow CR to use persistence
+
+Kubernetes's pods are stateless by definition. In some scenarios this can be a challenge for workloads that require to maintain the status of 
```

Comment:
```suggestion
Kubernetes's pods are stateless by definition. In some scenarios, this can be a challenge for workloads that require maintaining the status of 
```